app/presenters/concerns/OD2/generic_presenter_behavior.rb

```diff
@@ -0,0 +1,151 @@
+module OD2
+  module GenericPresenterBehavior
+    extend ActiveSupport::Concern
+    included do
+      delegate :alternative,
```

Comment:
Sadly, im not sure there is a good solution to this. However, it would be helpful if these were alphabetized too.
I think you could even use a quick ruby script to do it `[:alternative, ...].sort`